src/App.vue

```diff
@@ -225,9 +227,12 @@ export default class App extends Mixins(
 
     // Set base crumbs based on user role
     // Staff don't want the home landing page and they can't access the Manage Business Dashboard
-    if (this.isRoleStaff) {
-      // If staff, set StaffDashboard as home crumb
-      crumbs.unshift(getStaffDashboardBreadcrumb())
+    if (this.isAuthenticated) {
+      if (this.isRoleStaff) {
+        crumbs.unshift(getStaffDashboardBreadcrumb())
+      } else {
+        crumbs.unshift(getUserDashboardBreadcrumb())
```

Comment:
I think you don't need to do this.

If you're not logged in as staff, the breadcrumb link takes you to this page, which is titled "BC Registries Dashboard". It's the same page as now, but the text is wrong.

Reply:
The requirement changed. This conversation is obsolete.